task_process_berichten_in_confirmation.py

```diff
@@ -0,0 +1,80 @@
+import os
+from pytz import timezone
+from helpers import log
+
+from .sudo_query_helpers import query, update
+from .update_with_supressed_fail import update_with_suppressed_fail
+
+from .queries import STATUS_DELIVERED_CONFIRMED, STATUS_DELIVERED_UNCONFIRMED
+from .queries import construct_get_messages_by_status, construct_update_bericht_status
+from .queries import construct_create_kalliope_sync_error_query
+from .queries import construct_increment_confirmation_attempts_query
+
+from .kalliope_adapter import post_kalliope_poststuk_uit_confirmation
+from .kalliope_adapter import open_kalliope_api_session
+
+TIMEZONE = timezone('Europe/Brussels')
+MAX_CONFIRMATION_ATTEMPTS = int(os.environ.get('MAX_CONFIRMATION_ATTEMPTS'))
+PS_UIT_CONFIRMATION_PATH = os.environ.get('KALLIOPE_PS_UIT_CONFIRMATION_ENDPOINT')
+PUBLIC_GRAPH = "http://mu.semte.ch/graphs/public"
+
+
+def process_confirmations():
+    try:
+        log("Checking for new delivery confirmations to process")
+
+        query_string = construct_get_messages_by_status(STATUS_DELIVERED_UNCONFIRMED, MAX_CONFIRMATION_ATTEMPTS)
```

Comment:
I would favour to split the flow in two:
1. if max attempts is exceeded, set it to failed
2. else set to confirmed.
(I think in general we miss this for this service: a status failed, but that is for later)